cfgmgr/intfmgr.cpp

```diff
@@ -152,6 +158,14 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
         // Set Interface VRF except for lo
         if (!is_lo)
         {
+            /* Set nat zone */
+            if (!nat_zone.empty())
+            {
+                FieldValueTuple t("nat_zone", nat_zone);
+                fvVector.push_back(t);
+                m_appIntfTableProducer.set(alias, fvVector);
```

Comment:
set is at the end of outer if (!is_lo)

Reply:
This set it specific to nat_zone only, so using new "fvVector" variable for it.

Reply:
After resolved merge conflicts, doing set at the end of outer if(!is_lo).